elasticsearch/src/main/java/org/hibernate/search/elasticsearch/impl/ElasticsearchHSQueryImpl.java

```diff
@@ -328,13 +328,15 @@ private IndexSearcher() {
 			this.idFieldName = queriedEntityIdFieldName;
 
 			// Query filters; always a type filter, possibly a tenant id filter;
-			JsonObject effectiveFilter = getEffectiveFilter( typeFilters );
+			JsonObject filteredQuery = getFilteredQuery( jsonQuery.get( "query" ), typeFilters );
 
 			JsonBuilder.Object completeQuery = JsonBuilder.object();
-
-			completeQuery.add( "query",
-					JsonBuilder.object()
-							.add( "filtered", JsonBuilder.object( jsonQuery ).add( "filter", effectiveFilter ) ) );
+			// Copy the properties from the initial query
+			for ( Entry<String, JsonElement> entry : jsonQuery.entrySet() ) {
```

Comment:
Do we agree this particular thing is not related to the fix at hand? (just to be sure I understand correctly)